ai2thor/fifo_server.py

```diff
@@ -96,12 +99,28 @@ def _recv_message(self):
                 #print("body length %s" % len(body))
                 #print(body)
                 metadata = msgpack.loads(body, raw=False)
-                #print(metadata)
+            elif field_type is FieldType.METADATA_PATCH:
+                metadata_patch = msgpack.loads(body, raw=False)
+                agents = self.raw_metadata['agents']
```

Comment:
Can this be `None` the first time a message is received and is of type `FieldType.METADATA_PATCH`?

Reply:
That can never happen since sending the patched data always occurs after the sending of a full metadata.